sys/include/ztimer.h

```diff
@@ -488,11 +491,43 @@ void ztimer_update_head_offset(ztimer_clock_t *clock);
 void ztimer_init(void);
 
 /* default ztimer virtual devices */
+/**
+ * @brief   Base ztimer for the microsecond clock
+ */
+extern ztimer_clock_t *const ZTIMER_USEC_BASE;
+
 /**
  * @brief   Default ztimer microsecond clock
  */
 extern ztimer_clock_t *const ZTIMER_USEC;
 
+/**
+ * @def     ZTIMER_USEC_BASE_FREQ
+ *
+ * @brief   The frequency of the base ztimer for ZTIMER_USEC
+ */
+#ifdef XTIMER_HZ
+#define ZTIMER_USEC_BASE_FREQ   (XTIMER_HZ)
+#else
+#define ZTIMER_USEC_BASE_FREQ   (1000000LU)
+#endif /* XTIMER_HZ */
+
+/**
+ * @brief   Base ztimer for the millisecond clock
+ */
+extern ztimer_clock_t *const ZTIMER_MSEC_BASE;
+
+/**
+ * @def     ZTIMER_MSEC_BASE_FREQ
+ *
+ * @brief   The frequency of the base ztimer for ZTIMER_MSEC
+ */
+#ifdef MODULE_PERIPH_RTT
```

Comment:
This should also be `#ifdef MODULE_ZTIMER_PERIPH_RTT` to be congruent with the change in `ztimer/auto_init.c` line 108.

Reply:
done